src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/01 Executors/05 constant-arrival-rate.md

```diff
@@ -15,6 +15,14 @@ useful for a more accurate representation of RPS, for example.
 
 See the [arrival rate](/using-k6/scenarios/arrival-rate) section for details.
 
+<Blockquote mod="Note" title="Iteration starts are spaced fractionally">
+
+Iterations **do not** started at exactly the same time.
+At a `rate` of 10 with a `timeUnit` of `1s`, each iteration starts about every tenth of a second (that is, each 100ms).
+This jitter prevents accidental concurrency where all VUs start at the exact millisecond
```

Comment:
```suggestion
Iterations **do not** start at exactly the same time.
At a `rate` of `10` with a `timeUnit` of `1s`, each iteration starts about every tenth of a second (that is, each 100ms).
This jitter prevents accidental concurrency where all VUs start at the exact millisecond.
```

I am not actually sure about whether we should include the last sentence :thinking: the even spacing of iterations across the `timeUnit` interval is not exactly "jitter" and we might even add an option to group iteration starts in "buckets" (see 2nd point in https://github.com/grafana/k6/issues/1386)

Reply:
Sounds fine, will delete. I think "do not start at the same time" gets the point across.